saga/seata-saga-engine-store/src/main/java/io/seata/saga/engine/serializer/impl/ExceptionSerializer.java

```diff
@@ -75,25 +66,15 @@ public static Object deserializeByObjectInputStream(byte[] bytes) {
 
         Object result = null;
         if (bytes != null) {
-            ObjectInputStream ois = null;
-            try {
-                ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
-                ois = new ObjectInputStream(bais);
+            try (ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
+                ObjectInputStream ois = new ObjectInputStream(bais)) {
```

Comment:
The close method of ByteArrayInputStream actually does nothing.The close method of ObjectInputStream should be called.

Reply:
I see. I'll take care of that